src/toast/ops/sim_crosstalk.py

```diff
@@ -0,0 +1,480 @@
+# Copyright (c) 2015-2020 by the parties listed in the AUTHORS file.
+# All rights reserved.  Use of this source code is governed by
+# a BSD-style license that can be found in the LICENSE file.
+
+import numpy as np
+
+import traitlets
+
+from ..utils import Environment, Logger
+
+from ..timing import function_timer, Timer
+
+from ..noise_sim import AnalyticNoise
+
+from .. import rng
+from ..traits import trait_docs, Int, Unicode, Float, Bool, Instance, Quantity
+
+from .operator import Operator
+
+
+@function_timer
+def read_xtalk_matrix(filename, data):
+    """
+    Read Xtalk matrix from disc.
+    the  file to be in numpy binary format,
+    i.e. `*.npz`.
+
+    """
+
+    matrix = np.load(filename)["matrix"]
```

Comment:
In the future, we might consider using a different format where we have access to which row / column corresponds to a given detector, since we might have the same crosstalk matrix but different detector order in the observations (if some detectors are not active).